README.md

```diff
@@ -48,12 +48,16 @@ You must set at least one of `user_id_key` and `device_id_key`. They will be use
 #### time_key
 If set, `time_key` will be used to pull out a timestamp field to set as `time` in the Amplitude API request. This can be an array, the first matching key will be used.
 
+####  revenue_properties
+
+`revenue_properties` are a fixed set of properties (e.g., `price`, `quantity`, `revenue_type`) that Amplitude looks for in order to determine which events are revenue events (see the Amplitude HTTP API documentation for more information).
+
 #### user_properties and event_properties
 You can optionally specify lists of `user_properties` and `event_properties` to pull from the record.
 
 If `user_properties` are specified, only those properties will be included as `user_properties` in the Amplitude API call.  Otherwise no `user_properties` will be sent.
 
-If `event_properties` are specified, only those properties will be included as `event_properties` in the Amplitude API call. Otherwise the entire record (minus the key/value for `user_id_key` and `device_id_key`, and minus any `user_properties`) will be sent as `event_properties` to Amplitude.
+If `event_properties` are specified, only those properties will be included as `event_properties` in the Amplitude API call. Otherwise the entire record (minus the key/value for `user_id_key` and `device_id_key` and minus any `revenue_properties` and `user_properties`) will be sent as `event_properties` to Amplitude.
```

Comment:
nit: Wording is a little confusing.  Is this right?

If you don't specify `event_properties`, then everything with the exception of `user_id_key`, `device_id_key`, and anything specified in `revenue_properties` or `user_properties`, will be sent as `event_properties`.